Makefile

```diff
@@ -1,15 +1,14 @@
 GOTOOLS = \
 	github.com/mitchellh/gox \
 	github.com/Masterminds/glide \
-	github.com/tcnksm/ghr \
-	# gopkg.in/alecthomas/gometalinter.v2
-GOTOOLS_CHECK = gox glide ghr gometalinter.v2
+	github.com/tcnksm/ghr
+GOTOOLS_CHECK = gox glide ghr
 PACKAGES=$(shell go list ./... | grep -v '/vendor/')
 BUILD_TAGS?=tendermint
 TMHOME = $${TMHOME:-$$HOME/.tendermint}
 BUILD_FLAGS = -ldflags "-X github.com/tendermint/tendermint/version.GitCommit=`git rev-parse --short HEAD`"
 
-all: check build test install
+all: check build install test_integrations
```

Comment:
I think Jae wanted to run tests before install.

Reply:
the tests need the binary available on $PATH